engine/storage/datamotion/src/main/java/org/apache/cloudstack/storage/motion/AncientDataMotionStrategy.java

```diff
@@ -511,11 +517,16 @@ public void copyAsync(DataObject srcData, DataObject destData, Host destHost, As
                 answer = cloneVolume(srcData, destData);
             } else if (destData.getType() == DataObjectType.VOLUME && srcData.getType() == DataObjectType.VOLUME &&
                 srcData.getDataStore().getRole() == DataStoreRole.Primary && destData.getDataStore().getRole() == DataStoreRole.Primary) {
+                s_logger.info("About to MIGRATE copy between datasources");
```

Comment:
@rg9975 not sure if it was mentioned before but there are still some comments which are either INFO or DEBUG level. These can be TRACE level logs

Reply:
resolved in upcoming commit.